app/views/about/supporter_packt.html.haml

```diff
@@ -15,6 +15,7 @@
   .md-container
     %article
       .lhs
+        %h2.text-h2= @org.name
         %h2.text-h2 How Packt Publishing supports Exercism
         %hr.c-divider.mt-20.mb-28
         %p.text-p-xlarge.mb-40 Packt Publishing are a core Exercism partner, helping us develop both the fundraising and content side of the platform. They have sponsored our new Fundraising role, and we are currently developing a deeper partnership which we will launch in early 2022.
```

Comment:
So this paragraph should be replaced with `@org.support_description_html`

Reply:
And the paragraphs in the section below with `@org.description_html`

Reply:
@iHiD Which of these two fields is non-nullable? Right now, it is the `description`, but the `support_explanation` is shown on the https://exercism.org/about/supporters/organisations page

Reply:
And am I correct in assuming that we'll have organisations that have a full featured page (like packt and gobridge) and ones that don't (like Chicago)?